pkgs/development/interpreters/python/pypy/3/default.nix

```diff
@@ -0,0 +1,141 @@
+{ stdenv, substituteAll, fetchurl
+, zlib ? null, zlibSupport ? true, bzip2, pkgconfig, libffi
+, sqlite, openssl, ncurses, python, expat, tcl, tk, tix, xlibsWrapper, libX11
+, makeWrapper, callPackage, self, gdbm, db, lzma
+, python-setup-hook
+# For the Python package set
+, pkgs, packageOverrides ? (self: super: {})
+}:
+
+assert zlibSupport -> zlib != null;
+
+let
+  majorVersion = "5.10";
+  minorVersion = "1";
+  minorVersionSuffix = "";
+  pythonVersion = "3.5";
+  version = "${majorVersion}.${minorVersion}${minorVersionSuffix}";
+  libPrefix = "pypy${pythonVersion}";
+  sitePackages = "site-packages";
```

Comment:
```
sitePackages = "lib/${libPrefix}/site-packages";
```

Reply:
That doesn’t work (see main comment thread).